### DISCUSSION TITLE
Train and validate with my own data

### DISCUSSION THREAD

**Discussion by carmenrt** [DeepTrackAI/DeepTrack2#177] [Q&A] [answered]
Hello! I am a student of a master's degree in data analysis and systems biology and I am currently getting into the world of deep learning due to the topic of my master's thesis. Maybe it's a basic question, but how can I use the available notebooks to train and validate with my own dataset? Thank you very much in advance

**Comment by BenjaminMidtvedt** [comment, marked as answer]
Of course! You may want to play around with the parameters a bit to suit your needs better, but everything is designed to be easy to use with your own datasets easily
